tests/backup/backup_test.go

```diff
@@ -5403,19 +5405,20 @@ var _ = Describe("{ReplicaChangeWhileRestore}", func() {
 			ctx, err := backup.GetAdminCtxFromSecret()
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			for _, namespace := range bkpNamespaces {
-				backupName = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, namespace, time.Now().Unix())
-				restoreName = fmt.Sprintf("%s-%s-%v", restoreNamePrefix, backupName, time.Now().Unix())
-				scName := fmt.Sprintf("replica-sc-%v", time.Now().Unix())
-				pvcs, err := core.Instance().GetPersistentVolumeClaims(namespace, labelSelectors)
-				singlePvc := pvcs.Items[0]
-				dash.VerifyFatal(err, nil, "Getting PVC from namespace")
-				sourceScName, err := core.Instance().GetStorageClassForPVC(&singlePvc)
-				dash.VerifyFatal(err, nil, "Getting SC from PVC")
-				storageClassMapping[sourceScName.Name] = scName
-				restoredNameSpace := fmt.Sprintf("%s-%s", namespace, "restored")
-				namespaceMapping[namespace] = restoredNameSpace
-				err = CreateRestore(restoreName, backupName, namespaceMapping, SourceClusterName, orgID, ctx, storageClassMapping)
-				dash.VerifyFatal(err, nil, "Restoring with custom Storage Class Mapping")
+				for _, backupName := range backupNames {
+					restoreName = fmt.Sprintf("%s-%s-%v", restoreNamePrefix, backupName, time.Now().Unix())
+					scName := fmt.Sprintf("replica-sc-%v", time.Now().Unix())
+					pvcs, err := core.Instance().GetPersistentVolumeClaims(namespace, labelSelectors)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Getting all PVCs from namespace - %v", pvcs))
```

Comment:
this might be a long log if pvcs are many. just printing the namespace or number of pvcs should be fine.
you can use log.Infof for printing pvc names

Reply:
Done. Now printing only namespace and number of PVCs in it.